docs/gpu_performance_tips.md

```diff
@@ -475,6 +476,25 @@ def main(_):
   output_buffer = entry_computation(weights, input_buffer, mesh)
   print(f"output_buffer = \n{output_buffer}")
 ```
+
+#### Using `psend` and `precv`
```

Comment:
@frgossen FYI I added documentation here

Reply:
Let's add this after some users try it out and we fix all the bugs.

Reply:
Sounds good. Will send out a separate PR shortly.